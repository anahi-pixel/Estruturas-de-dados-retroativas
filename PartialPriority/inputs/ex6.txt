1 70 700
1 50 500
1 60 600
1 10 300
1 40 400
1 80 800
1 30 100
5
6
2 20
5
6
2 35
5
6
2 55
7
5
6
1 65 150
7
5
6