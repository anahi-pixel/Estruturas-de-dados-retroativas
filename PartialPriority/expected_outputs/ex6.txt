100
7
100
6
400
5
1 Fila de prioridade no instante atual: 
500 600 700 800 
500
4
2 Fila de prioridade no instante atual: 
150 500 600 700 800 
150
5
